### ISSUE TITLE
After pressing escape key to close the information modal window, outline is not visible on currently focused button

### ISSUE THREAD

**Issue by manuellagrand** [readium/readium-desktop#425]
outline is not visible after pressing escape key to close the information modal window.

![image.png](https://images.zenhubusercontent.com/5d19d4e1599844363331bf70/5a1819f1-0df6-4f9a-b9b8-cb689d504cfe)

focus should come back to the information button.

![image.png](https://images.zenhubusercontent.com/5d19d4e1599844363331bf70/4605868a-1d97-4e88-a6e8-4690cc6ba329)

currently the focus is lost.

**Comment by danielweck** [comment]
Yes, same issue when opening the book info dialog from the library / bookshelf view. Normally, keyboard focus should return on the calling site which in this case is a popup menu item ... but because the menu is hidden, perhaps it would be better to focus on the calling site of the menu itself, which is the vertical dotted button.

![ThoriumDialogKeyboardFocus](https://user-images.githubusercontent.com/625201/61203814-642d7b00-a6e3-11e9-9ae0-3a0433f13813.png)
